tool/tctl/common/resource_command.go

```diff
@@ -68,71 +71,77 @@ Same as above, but using JSON output:
 `
 
 // Initialize allows ResourceCommand to plug itself into the CLI parser
-func (g *ResourceCommand) Initialize(app *kingpin.Application, config *service.Config) {
-	g.CreateHandlers = map[ResourceKind]ResourceCreateHandler{
-		services.KindUser:            g.createUser,
-		services.KindTrustedCluster:  g.createTrustedCluster,
-		services.KindGithubConnector: g.createGithubConnector,
-		services.KindCertAuthority:   g.createCertAuthority,
+func (rc *ResourceCommand) Initialize(app *kingpin.Application, config *service.Config) {
+	rc.CreateHandlers = map[ResourceKind]ResourceCreateHandler{
+		services.KindUser:            rc.createUser,
+		services.KindTrustedCluster:  rc.createTrustedCluster,
+		services.KindGithubConnector: rc.createGithubConnector,
+		services.KindCertAuthority:   rc.createCertAuthority,
 	}
-	g.config = config
+	rc.config = config
+
+	rc.createCmd = app.Command("create", "Create or update a Teleport resource from a YAML file")
+	rc.createCmd.Arg("filename", "resource definition file").Required().StringVar(&rc.filename)
+	rc.createCmd.Flag("force", "Overwrite the resource if already exists").Short('f').BoolVar(&rc.force)
 
-	g.createCmd = app.Command("create", "Create or update a Teleport resource from a YAML file")
-	g.createCmd.Arg("filename", "resource definition file").Required().StringVar(&g.filename)
-	g.createCmd.Flag("force", "Overwrite the resource if already exists").Short('f').BoolVar(&g.force)
+	rc.deleteCmd = app.Command("rm", "Delete a resource").Alias("del")
+	rc.deleteCmd.Arg("resource type/resource name", `Resource to delete
```

Comment:
The multiline string here is pretty hacky but works. The only other way I could see to add 'Examples' to command output would be to start modifying `kingpin` to support it.